### ISSUE TITLE
Update a11y documentation

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2553]
### Subject of the issue
Adapt should be able to read "$5bn" as "five billion dollars". This is a slightly different use-case to abbreviations which should be pronounced correctly whereas here an entirely alternative text should be read.

Can be solved with a helper.

**Comment by moloko** [comment]
Could we have an option to make it read [like this](https://youtu.be/jTmXHvGZiSY?t=33)?

**Comment by moloko** [comment]
Reminder for one of us to update the a11y documentation to include something about this in the a11y wiki when released.

Have a feeling that the zero-width space stuff might also need writing up...?

**Comment by oliverfoster** [comment]
Also the helpers need updating in the WIKI

**Comment by kirsty-hames** [comment]
> Reminder for one of us to update the a11y documentation to include something about this in the a11y wiki when released.
> 
> Have a feeling that the zero-width space stuff might also need writing up...?

`a11y_alt_text` documented [here on the wiki ](https://github.com/adaptlearning/adapt_framework/wiki/Accessibility-v3-%E2%86%92-v4#screen-reader-specific-text)but zero-width needs adding.

**Comment by oliverfoster** [comment]
duplicate of https://github.com/adaptlearning/adapt_framework/issues/3379
